kernel-install/50-mkosi.install

```diff
@@ -153,20 +154,21 @@ def main() -> None:
         "--output-dir", context.staging_area,
     ]  # fmt: skip
 
+    if context.kernel_image:
+        cmdline += ["--kernel-image", context.kernel_image]
+
     if context.verbose:
         cmdline += ["--debug"]
 
     logging.info(f"Building {output}")
 
-    run(cmdline)
-
-    (context.staging_area / output).unlink()
+    run(cmdline, stdin=sys.stdin, stdout=sys.stdout)
 
     if format == OutputFormat.cpio:
         build_microcode_initrd(context.staging_area / "microcode")
     else:
-        (context.staging_area / f"{output}.vmlinuz").unlink()
-        (context.staging_area / f"{output}.initrd").unlink()
+        (context.staging_area / f"{output}.vmlinuz").unlink(missing_ok=True)
+        (context.staging_area / f"{output}.initrd").unlink(missing_ok=True)
```

Comment:
This can just be dropped and instead you can add `SplitArtifacts=` to `mkosi-initrd/mkosi.conf` in the `[Output]` section.

Reply:
but why should it fail if it's not there? It doesn't seem to be used